### PR TITLE
source-hubspot: migrate to base image

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Migrating `source-hubspot` myself as I noticed the `build` fails when using our new base image.

## How
`airbyte-ci connectors --name=source-hubspot migrate_to_base_image 31644`

I fixed the build by removing the `requests` explicit dependency. This connector will get `requests` from the CDK deps.

We have to wait for https://github.com/airbytehq/airbyte/pull/31606 to be mainlined to fix CAT run

### CODE REVIEW

**Comment by alafanechere** [comment]
Current dependencies on/for this PR:
* master
  * **PR #31644** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/31644" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/31644?utm_source=stack-comment).

File: airbyte-integrations/connectors/source-hubspot/setup.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
     "airbyte-cdk",
     "backoff==1.11.1",
     "pendulum==2.1.2",
-    "requests==2.26.0",
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-hubspot/setup.py:12]: `requests` is already an `airbyte-cdk` transitive dependency.
Pinning it to an "old" 2.26.0 version leads to the following error when running `spec`: 
```
│ │     ┃ Traceback (most recent call last):
│ │     ┃   File "/airbyte/integration_code/main.py", line 8, in <module>
│ │     ┃     from airbyte_cdk.entrypoint import launch
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/__init__.py", line 6, in <module>
│ │     ┃     from .entrypoint import AirbyteEntrypoint
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 17, in <module>
│ │     ┃     import requests
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/requests/__init__.py", line 100, in <module>
│ │     ┃     check_compatibility(urllib3.__version__, chardet_version, charset_normalizer_version)
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/requests/__init__.py", line 85, in check_compatibility
│ │     ┃     raise Exception("You need either charset_normalizer or chardet installed")
│ │     ┃ Exception: You need either charset_normalizer or chardet installed
``` 
Remove `requests` from `setup.py` ensure the connector will use whatever version the CDK needs. The request's CDK version does not lead to this error when it's built inside or python base image.

Comment (bazarnov) [reply airbyte-integrations/connectors/source-hubspot/setup.py:12]: I'm going to fix this here: https://github.com/airbytehq/airbyte/pull/31606

Comment (alafanechere) [reply airbyte-integrations/connectors/source-hubspot/setup.py:12]: @bazarnov let me know when https://github.com/airbytehq/airbyte/pull/31606 is merged and I'll update this branch

**Comment by bazarnov** [comment]
@alafanechere final CI checks for this one https://github.com/airbytehq/airbyte/pull/31606 and we're good to go. I've also cleared the `main` dependencies in `setup.py`.

**Review by bazarnov — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  source-paypal-transaction test report (commit [`e6039482d1`](https://github.com/airbytehq/airbyte/pull/31644/commits/e6039482d13f0c1a5359f775237c75a269efc79c)) - ✅

⏲️  Total pipeline duration: 01mn08s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Build source-paypal-transaction docker image for platform(s) linux/x86_64 | ✅       |
| Acceptance tests                                                          | ✅       |
| Code format checks                                                        | ✅       |
| Validate metadata for source-paypal-transaction                           | ✅       |
| Connector version semver check                                            | ✅       |
| QA checks                                                                 | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/augustin_10-20-source-hubspot_migrate_to_base_image/1697794439/e6039482d13f0c1a5359f775237c75a269efc79c/source-paypal-transaction/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:e6039482d13f0c1a5359f775237c75a269efc79c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-paypal-transaction test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-hubspot test report (commit [`e6039482d1`](https://github.com/airbytehq/airbyte/pull/31644/commits/e6039482d13f0c1a5359f775237c75a269efc79c)) - ❌

⏲️  Total pipeline duration: 08mn37s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-hubspot docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Integration tests                                              | ✅       |
| Acceptance tests                                               | ❌       |
| Code format checks                                             | ✅       |
| Validate metadata for source-hubspot                           | ✅       |
| Connector version semver check                                 | ✅       |
| Connector version increment check                              | ✅       |
| QA checks                                                      | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/augustin_10-20-source-hubspot_migrate_to_base_image/1697794439/e6039482d13f0c1a5359f775237c75a269efc79c/source-hubspot/1.5.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:e6039482d13f0c1a5359f775237c75a269efc79c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by bazarnov** [comment]
@alafanechere It's merged.

**Comment by airbyte-oss-build-runner** [comment]
##  source-hubspot test report (commit [`78bdcb0594`](https://github.com/airbytehq/airbyte/pull/31644/commits/78bdcb059468c56bdd3b9c3fecacd36e49dd4d0c)) - ❌

⏲️  Total pipeline duration: 12mn13s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-hubspot docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Integration tests                                              | ✅       |
| Acceptance tests                                               | ✅       |
| Code format checks                                             | ✅       |
| Validate metadata for source-hubspot                           | ✅       |
| Connector version semver check                                 | ✅       |
| QA checks                                                      | ❌       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/augustin_10-20-source-hubspot_migrate_to_base_image/1697796863/78bdcb059468c56bdd3b9c3fecacd36e49dd4d0c/source-hubspot/1.6.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:78bdcb059468c56bdd3b9c3fecacd36e49dd4d0c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-hubspot test report (commit [`fd1a4790b7`](https://github.com/airbytehq/airbyte/pull/31644/commits/fd1a4790b7036aaeed0dc11b84a2301ba0448cf9)) - ✅

⏲️  Total pipeline duration: 13mn19s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-hubspot docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Integration tests                                              | ✅       |
| Acceptance tests                                               | ✅       |
| Check our base image is used                                   | ✅       |
| Code format checks                                             | ✅       |
| Validate metadata for source-hubspot                           | ✅       |
| Connector version semver check                                 | ✅       |
| Connector version increment check                              | ✅       |
| QA checks                                                      | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/augustin_10-20-source-hubspot_migrate_to_base_image/1697797815/fd1a4790b7036aaeed0dc11b84a2301ba0448cf9/source-hubspot/1.6.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:fd1a4790b7036aaeed0dc11b84a2301ba0448cf9)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Pull request merged**